code/mobility-services/mobility-services.css

```diff
@@ -46,14 +46,23 @@
 }
 
 /* This is for the bars */
-.progressbar li{
+.progressbar li{ /* Chauffeur Permit Form */
```

Comment:
Might be worth moving these comments to new lines above as section headers so its easier to find each section